docs/content_management/configure_ct_field_settings.md

```diff
@@ -0,0 +1,98 @@
+---
+description: Configure Field settings in Content Types.
+---
+
+# Configure Content Type Fields
+
+When you [modify a Content Type](create_edit_content_types.md) and add or modify Fields, 
```

Comment:
You can think about adding link to [Fields](../content_management/content_model.md#fields-and-field-types)